src/Plugins/LevelDBStore/Plugins/Storage/Store.cs

```diff
@@ -9,59 +9,65 @@
 // Redistribution and use in source and binary forms with or without
 // modifications are permitted.
 
-using Neo.IO.Data.LevelDB;
+using Neo.IO.Storage.LevelDB;
 using Neo.Persistence;
+using System.Collections;
 using System.Collections.Generic;
 
 namespace Neo.Plugins.Storage
 {
-    internal class Store : IStore
+    internal class Store : IStore, IEnumerable<KeyValuePair<byte[], byte[]>>
     {
-        private readonly DB db;
+        private readonly DB _db;
 
-        public Store(string path)
+        public Store(string dir)
         {
-            db = DB.Open(path, new Options { CreateIfMissing = true, FilterPolicy = Native.leveldb_filterpolicy_create_bloom(15) });
+            _db = new(dir, new()
+            {
+                CreateIfMissing = true,
+                // Keep whole blockchain open plus future files
+                // at lease up to block index 10_000_000
+                MaxOpenFiles = 4096,
+                FilterPolicy = 10,
```

Comment:
```suggestion
                FilterPolicy = Native.leveldb_filterpolicy_create_bloom(15),
```

?

Reply:
No need for extra, 5 bytes. Bloom filters should be as small as possible for datasets without that many collisions. 10 is way more than enough.